src/components/SingleProduct/Details.js

```diff
@@ -3,8 +3,18 @@ import { Stack, Typography } from "@mui/material";
 import Button from "../Login/Button";
 import CircleIcon from "@mui/icons-material/Circle";
 import { DetailsItem } from "../../StyledComponents";
+import AddCircleIcon from "@mui/icons-material/AddCircle";
+import RemoveCircleIcon from "@mui/icons-material/RemoveCircle";
+import { useState } from "react";
 function Details(props) {
+  const [quantity, setQuantity] = useState(0);
```

Comment:
initial quantity should be always 1, please update the initial state value

Reply:
done